src/client/gov/mod.rs

```diff
@@ -131,4 +139,63 @@ impl Contact {
         self.send_message(&[msg], None, &[fee], wait_timeout, private_key)
             .await
     }
+
+    /// Encodes and submits a proposal change bridge parameters, should maybe be in deep_space
+    pub async fn submit_parameter_change_proposal(
+        &self,
+        proposal: ParameterChangeProposal,
+        deposit: Coin,
+        fee: Coin,
+        key: impl PrivateKey,
+        wait_timeout: Option<Duration>,
+    ) -> Result<TxResponse, CosmosGrpcError> {
+        // encode as a generic proposal
+        let any = encode_any(proposal, PARAMETER_CHANGE_PROPOSAL_TYPE_URL.to_string());
+        self.create_gov_proposal(any, deposit, fee, key, wait_timeout)
+            .await
+    }
+
+    /// Encodes and submits a proposal to upgrade chain software, should maybe be in deep_space (sorry)
+    pub async fn submit_upgrade_proposal(
+        &self,
+        proposal: SoftwareUpgradeProposal,
+        deposit: Coin,
+        fee: Coin,
+        key: impl PrivateKey,
+        wait_timeout: Option<Duration>,
+    ) -> Result<TxResponse, CosmosGrpcError> {
+        // encode as a generic proposal
+        let any = encode_any(proposal, SOFTWARE_UPGRADE_PROPOSAL_TYPE_URL.to_string());
+        self.create_gov_proposal(any, deposit, fee, key, wait_timeout)
+            .await
+    }
+    /// Encodes and submits a proposal to register a Coin for use with the ev module
+    pub async fn submit_register_coin_proposal(
```

Comment:
These two Register*Proposals we're likely to submit a lot, if we end up making a rust-based CLI for althea chain, these sorts of functions will be really useful.